src/content/docs/pt-br/basics/astro-components.mdx

```diff
@@ -223,16 +223,16 @@ const { titulo } = Astro.props;
 ---
 <div id="invólucro-do-conteúdo">
   <Cabecalho />
-  <slot name="depois-do-cabecalho"/>  <!--  filhos com o atributo `slot="depois-do-cabecalho"` irão para cá -->
+  <slot name="depois-do-cabecalho"/>  <!--  filhos com o atributo `slot="depois-do-cabecalho"` virão para cá -->
   <Logo />
   <h1>{titulo}</h1>
-  <slot />  <!--  filhos sem um atributo `slot`, ou com `slot="default"` irão para cá -->
+  <slot />  <!--  filhos sem um atributo `slot`, ou com `slot="default"` virão para cá -->
   <Rodape />
-  <slot name="depois-do-rodape"/>  <!--  filhos com o atributo `slot="depois-do-rodape"` irão para cá -->
+  <slot name="depois-do-rodape"/>  <!--  filhos com o atributo `slot="depois-do-rodape"` virão para cá -->
```

Comment:
"Irão para cá" is contradictory; "irão" implies moving somewhere else (not here), "cá" implies here.